src/internal/packager/helm/post-render.go

```diff
@@ -32,86 +31,71 @@ import (
 )
 
 type renderer struct {
-	*Helm
+	chart v1alpha1.ZarfChart
+
+	adoptExistingResources bool
+	cluster                *cluster.Cluster
```

Comment:
Does renderer need to mutate `cluster.Cluster`, `types.ZarfState`, `action.Configuration`, and `variables.VariableConfig` or can we pass these in as copies?